scripts/project-setup-scripts/add-folder-roles-to-service-accounts.sh

```diff
@@ -0,0 +1,36 @@
+# set up folder access for Rawls and Cromwell SAs as editor roles
+# must have resourcemanager.projects.setIamPolicy permissions
```

Comment:
Do you mean that the person running this script needs this permission?  Maybe reword this as something like:

```suggestion
# In order to successfully run this script, you must first `gcloud auth login` as a gcloud user with resourcemanager.projects.setIamPolicy permissions
```